docs/admin/service-accounts-admin.md

```diff
@@ -96,13 +96,14 @@ account, and the controller will update it with a generated token:
 ```json
 secret.json:
 {
+    "apiVersion": "v1",
```

Comment:
You know what's more weird, `liggitt was unassigned by dalanlan 8 hours ago`.
I was totally offline that time and i dont have the rights without doubt. Sorry for the trouble.